pkg/kubelet/remote/remote_runtime.go

```diff
@@ -303,6 +306,9 @@ func (r *RemoteRuntimeService) ContainerStatus(containerID string) (*runtimeapi.
 			klog.Errorf("ContainerStatus of %q failed: %v", containerID, err)
 			return nil, err
 		}
+		if resp.Status.GetReason() == "OOMKilled" {
+			metrics.RemoteContainerOOMCounter.WithLabelValues("").Inc()
```

Comment:
with a non-counter-vec counter, no need for WithLabelValues

Reply:
Is there a reason why we are blanking the operation type here?